<?xml version="1.0" encoding="utf-8"?>
<!--launch file for smoke package.-->
<launch>

  <!-- ROS and network parameter files -->
  <arg name="ros_param_file" default="$(find smoke)/config/darknet_svm.yml"/>

  <!-- Load parameters -->
  <rosparam command="load" ns="smoke" file="$(arg ros_param_file)"/>

  <!--robot action-->
  <!--node pkg="smoke" type="kobuki_actTest.py" name="kobuki_actclient_node" output="screen" /-->
  <node pkg="smoke" type="action_buttons.py" name="kobuki_buttons_node" output="screen" />
  
  <!--robot action-->
  <node pkg="smoke" type="kobuki_actSrv.py" name="kobuki_action_node" output="screen" />
</launch>
